sumologic-service/0.15.0/README.md

```diff
@@ -0,0 +1,151 @@
+```
+Example:
+```bash
+keptn add-resource --project="podtatohead" --stage="hardening" --service="helloservice" --resource=./quickstart/sli.yaml --resourceUri=sumologic/sli.yaml
+keptn add-resource --project="podtatohead" --stage="hardening" --service="helloservice" --resource=./quickstart/slo.yaml --resourceUri=slo.yaml
+```
+Check [./quickstart/sli.yaml](https://raw.githubusercontent.com/keptn-sandbox/sumologic-service/release-0.15.0/examples/quickstart/sli.yaml) and [./quickstart/slo.yaml](https://raw.githubusercontent.com/keptn-sandbox/sumologic-service/release-0.15.0/examples/quickstart/slo.yaml) for example SLI and SLO. 
+
+<!-- TODO: Uncomment this after the PR to support switching SLI provider is merged -->
+<!-- 4. Configure Keptn to use Sumo Logic SLI provider
+Use keptn CLI version [0.15.0](https://github.com/keptn/keptn/releases/tag/0.15.0) or later.
+```bash
+keptn configure monitoring sumologic --project <project-name>  --service <service-name>
+``` -->
+
+5. Trigger delivery
+```bash
+keptn trigger delivery --project=<project-name> --service=<service-name> --image=<image> --tag=<tag>
+```
+Example:
+```bash
+keptn trigger delivery --project=podtatohead --service=helloservice --image=docker.io/jetzlstorfer/helloserver --tag=0.1.1
+```
+Observe the results in the [Keptn Bridge](https://keptn.sh/docs/0.15.x/bridge/)
+
+# Not supported in the query
+- `fillmissing`
+- `outlier`
+- `timeshift`
+
+Why? Because the API does not support `fillmissing` and `outlier`. `timeshift` is supported but you can't write it in the query like `<my-query> | timeshift`. We plan to support `timeshift` in the future ([issue](https://github.com/vadasambar/sumologic-service/issues/1)) but support for `fillmissing` and `outlier` depends on Sumo Logic (can't do anything until Sumo Logic supports it). 
+
+# Rules for using `quantize`
+Based on https://help.sumologic.com/Metrics/Metric-Queries-and-Alerts/07Metrics_Operators/quantize#quantize-syntax
+1. Use only 1 `quantize` (using `quantize` multiple times in a query leads to error)
+2. Use `quantize` immediately after the metric query before any other operator
+3. Quantize should be strictly defined as `query | quantize to [TIME INTERVAL] using [ROLLUP]` (this differs from how Sumo Logic quantize works. You need to be explicit here. Dropping [TIME INTERVAL] or `using` or `[ROLLUP]` won't work)  
+
+Why so many rules? Because [Sumo Logic API does not support quantize in the query](https://api.sumologic.com/docs/#operation/runMetricsQueries). We have implemented a wrapper
+which mimics quantize which works well if you adhere to 
+
+## Compatibility Matrix
+
+*Please fill in your versions accordingly*
+
+| Keptn Version    | [sumologic-service Docker Image](https://github.com/keptn-sandbox/sumologic-service/pkgs/container/sumologic-service) |
+|:----------------:|:----------------------------------------:|
+|       0.15.0      | keptn-sandbox/sumologic-service:0.15.0 |
```

Comment:
Is this service tested and supported against any new Keptn versions like 0.17 or 0.18?

Reply:
Not as of now but I plan to support more recent versions.

Reply:
Removed a line from the boilerplate (not needed)